Editors/ImportExportEditor/Editors.ImportExport/Exporting/Exporters/RmvToGltf/Helpers/GltfAnimationBuilder.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Automation;
+using Editors.ImportExport.Common;
+using Editors.Shared.Core.Services;
+using GameWorld.Core.Animation;
+using Shared.Core.PackFiles;
+using Shared.Core.PackFiles.Models;
+using Shared.GameFormats.Animation;
+using SharpGLTF.Schema2;
+using SysNum = System.Numerics;
+
+namespace Editors.ImportExport.Exporting.Exporters.RmvToGltf.Helpers
+{
+    public class GltfAnimationBuilder
+    {
+        private readonly IPackFileService _packFileService;
+        private readonly SkeletonAnimationLookUpHelper _skeletonLookUpHelper;
+
+        public GltfAnimationBuilder(IPackFileService packFileServoce, SkeletonAnimationLookUpHelper skeletonLookUpHelper)
+        {
+            _packFileService = packFileServoce;
+            _skeletonLookUpHelper = skeletonLookUpHelper;
+        }
+
+        public void Build(string skeletonName, RmvToGltfExporterSettings settings, ProcessedGltfSkeleton gltfSkeleton, ModelRoot outputScene)
+        {
+            if (settings.ExportAnimations == false)
```

Comment:
This check should be outside of the function. If we are not exporting animation, we dont want to call the function. 
(Makes testing the future eaiser as we can then verify if the function was called or not based on the setting. If the setting is inside, we can not use that approach)